Add look and feel page (#66)
hyphacoop/handbook: look-and-feel.md

# Look and Feel

> ### Name and wordmark

Reviewer: How come we skipped a level?

Author: format to size

Our name uses title case with the following variations:

> **HYPHA**  
> **HYPHA** Worker Co-operative  
> Hypha Worker Co-operative Inc. [our legal name]  

> We use the following wordmark in online spaces, services and social media where we represent the organization (e.g., [Twitter](https://twitter.com/hyphacoop) and [LinkedIn](https://ca.linkedin.com/company/hyphacoop)). 

Reviewer 2: `data-proofer-ignore` attr could go on the LinkedIn anchor to get around the proofer error.

Author: gitbook uses a custom parser based on kramdown from what I can tell (most docs disappearing), so I can't just pass this with the {: } syntax. I'm resolving this problem by moving where the content is (not essential here anyway). :wink:

<img src="images/hypha-wordmark.png" alt="Hypha Wordmark in white on purple background" width="50%"/>

### Typography

> Our **heading typeface** is [Work Sans](https://fonts.google.com/specimen/Work+Sans) and our **secondary (body) typeface** is !!!!. For the web we prefer using the [system font stack](https://css-tricks.com/snippets/css/system-font-stack/) sans-serif typeface.

Reviewer 2: Proxima Nova could be listed as a secondary typeface for now if we go by the deck

Author: Just realized this isn't a free font :flushed:

> Our **heading typeface** is [Work Sans](https://fonts.google.com/specimen/Work+Sans) and our **secondary (body) typeface** is !!!!. For the web we prefer using the [system font stack](https://css-tricks.com/snippets/css/system-font-stack/) sans-serif typeface.

Reviewer 2: ```suggestion
Our **heading typeface** is [Work Sans](https://github.com/weiweihuanghuang/Work-Sans) and our **secondary (body) typeface** is !!!!. For the web we prefer using the [system font stack](https://css-tricks.com/snippets/css/system-font-stack/) sans-serif typeface.
```

Heading typeface 
>

Body typeface 
>

Body typeface (web)
> `font-family: -apple-system, BlinkMacSystemFont, avenir next, avenir, helvetica neue, helvetica, ubuntu, roboto, noto, segoe ui, arial, sans-serif`

### Colours

> <img src="images/hypha-colours.png" alt="Hypha colours" width="90%"/>

Reviewer 2: Could the text on the Lime swatch be translucent black?

> Registration Black   

Reviewer 2: I associate "Registration Black" to the CMYK process black. I think X11 is just "Black"

Author: Okay, was just using the names that I was told were associated!

`#000000;`  
`rgb(0, 0, 0);`  

Grey   
`#777777;`   
`rgb(119, 119, 119);`

White  
`#FFFFFF;`  
`rgb(255, 255, 255);`  
  
Vivid Violet  
`#9900FC;`  
`rgb(153, 0, 252);`  

Lime (Web) (X11 Green)  
`#00ff00;`  
`rgb(0, 255, 0);`  

### Preferred icons and emoji

We tend to use the following emoji and iconography: 

🍄 `:mushroom:`  

> ⚑  

Reviewer 2: ```suggestion
⚑  `&#9873;`
```

Reviewer 2: Maybe include the 🏴 emoji as well?

Author: Sure, if we want to be anarchists about it ;)

> ### Assets and Templates

Reviewer 2: ```suggestion
### Assets and templates
```

- Wordmark
- Image bank
- Letterhead 

> - Report Template

Reviewer 2: ```suggestion
- Report template
```
